controllers/commoncode.go

```diff
@@ -162,6 +162,7 @@ func (r *BestieReconciler) updateApplicationStatus(ctx context.Context, bestie *
 		client.InNamespace(bestie.Namespace),
 		client.MatchingLabels{"app": "bestie"},
 	}
+  
```

Comment:
this file needs gofmt .. like yuri said need to run `gofmt -w controllers/commoncode.go` to fix the linting error